index.js

```diff
@@ -53,32 +76,33 @@ const displayBooks = () => {
   saveData();
 };
 
-/* form functions */
-const form = document.querySelector('form');
-const author = form.querySelector('#author');
-const title = form.querySelector('#title');
-
 const getData = () => {
   const formData = JSON.parse(localStorage.getItem('myBooks'));
   if (formData == null) {
-    books = [];
+    methods.books = [];
   } else {
-    books = formData;
+    methods.books = formData;
   }
 };
 
-window.onload = getData();
+// window.onload = getData();
 window.onbeforeunload = () => {
   getData();
   displayBooks();
 };
 
+/* form functions */
+const form = document.querySelector('form');
+const author = form.querySelector('#author');
+const title = form.querySelector('#title');
+
 const addBtn = document.querySelector('#add-btn');
 addBtn.addEventListener('click', (e) => {
   e.preventDefault();
   const bookTitle = title.value;
   const bookAuthor = author.value;
-  addBook(bookTitle, bookAuthor);
+  const bookId = methods.books.length + 1;
```

Comment:
- [ ] I will recommend you to use some random number auto-generated to use as `id`, I don't recommend you to use the length of the array as `id`, kindly change this.
     - And for the `removeBook` function, you could use `.filter()` to find and remove the book